frontend/src/components/Pages/RobotPage/LocalizationDialog.tsx

```diff
@@ -97,8 +97,7 @@ export const LocalizationDialog = ({ robot }: RobotProps): JSX.Element => {
 
     const onClickLocalize = () => {
         if (selectedAssetDeck && localisationPose) {
-            BackendAPICaller.postLocalizationMission(localisationPose, robot.id)
-        }
```

Comment:
Done. Thanks !

Reply:
@eivindsjovold  can you please review again since you chose to request changes and not simply review, now you need to review until approved for this to go in. Thanks !